packaging/tags.py

```diff
@@ -407,7 +407,12 @@ def _mac_binary_formats(version, cpu_arch):
             return []
         formats.extend(["fat32", "fat"])
 
-    formats.append("universal")
+    elif cpu_arch == "arm64":
+        formats.append("universal2")
+
+    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc"}:
+        formats.append("universal")
```

Comment:
Would it be worth doing the "universal2" addition the same as you do for "universal" below (with the appropriate changes above)?

```suggestion
    if cpu_arch in {"arm64", "x86_64"}:
        formats.append("universal2")
        
    if cpu_arch in {"x86_64", "i386", "ppc64", "ppc"}:
        formats.append("universal")
```